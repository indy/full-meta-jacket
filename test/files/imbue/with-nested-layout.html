---
"template": "nested-layout-2"
---
<p>Here's some more text</p>
