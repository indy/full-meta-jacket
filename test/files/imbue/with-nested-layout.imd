---
"template": "nested-layout-2"
---
Here's some more text that should be parsed as markdown
