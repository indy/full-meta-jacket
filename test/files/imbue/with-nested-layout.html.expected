<html><body><h1>hello world</h1> 
<p>Here's some more text</p>
 </body></html>
